lib/ramble/ramble/workspace/workspace.py

```diff
@@ -62,7 +62,14 @@
 _active_workspace = None
 
 #: Path where workspaces are stored
-workspace_path = os.path.join(ramble.paths.var_path, 'workspaces')
+workspace_path = ramble.config.get('config:user:workspace_dir')
+if workspace_path:
+    try:
```

Comment:
Given this try block, maybe this warrants a test to ensure we're trapping what we think we are.

Reply:
Outdated? I tried a different way of setting it.